addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/handler/OnkyoHandler.java

```diff
@@ -444,6 +480,28 @@ public void statusUpdateReceived(String ip, EiscpMessage data) {
         }
     }
 
+    private void processInfo(String infoXML) {
+
+        try {
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder builder;
+            builder = factory.newDocumentBuilder();
+            InputSource is = new InputSource(new StringReader(infoXML));
+            Document doc = builder.parse(is);
+
+            NodeList SelectableInputs = doc.getDocumentElement().getElementsByTagName("selector");
+            this.populateInputs(SelectableInputs);
```

Comment:
This can be removed

Reply:
Okay, I'll remove the `this` reference on this item and `processInfo`